source/03-components/accordion/accordion.stories.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
This line isn’t needed since we’re not using `React` in this file.

Reply:
Removed